portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -886,8 +882,7 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LPS-105156. Authorized application should be automatically removed from the User's connect table when it is expired. Blocked by LPS-191344."
-	@ignore = "true"
+	@description = "This is a use case for LPS-105156. Authorized application should be automatically removed from the User's connect table when it is expired."
```

Comment:
This test case still failed at CI run, this one should remain in quarantine while we make it more stable

Reply:
Ok  and done